crates/daphne-server/src/lib.rs

```diff
@@ -100,6 +112,38 @@ impl router::DaphneService for App {
     fn signing_key(&self) -> Option<&p256::ecdsa::SigningKey> {
         self.service_config.signing_key.as_ref()
     }
+
+    fn check_bearer_token(
+        &self,
+        token: &BearerToken,
+        sender: DapSender,
+        task_id: Option<TaskId>,
+    ) -> Result<(), String> {
```

Comment:
This return type looks strange to me. I guess`Ok(())` means the token is good and `Err(s)` means it's bad? I'd suggest documenting the behavior of this function.

Reply:
I did document, but I documented in the trait and not in the implementation